reactors/1D_packbed.ipynb

```diff
@@ -0,0 +1,518 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# One-dimensional packed-bed, catalytic-membrane reactor model without streamwise diffusion"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The present model simulates heterogeneous catalytic processes inside packed-bed, catalytic membrane reactors. The gas-phase and surface-phase species conservation equations are derived and the system of differential-algebraic equations (DAE) is solved using the scikits.odes.dae IDA solver."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Runnning Cantera version: 2.5.1\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Import Cantera and scikits\n",
+    "import numpy as np\n",
+    "from scikits.odes import dae\n",
+    "import cantera as ct\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "%matplotlib inline\n",
+    "print(\"Runnning Cantera version: \" + ct.__version__)"
```

Comment:
The other notebooks have been changed to use the SVG backend of matplotlib, I'm pretty sure. Can you check and change this over? Can you also run with Cantera 2.6.0?

Reply:
I have added the SVG backend. Also running it on 2.6.0 now.